src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1018,16 +995,15 @@
   (let [csr-path (path-to-cert-request csrdir subject)]
     (if (fs/exists? csr-path)
       (if (fs/delete csr-path)
-        (let [msg (str "Deleted certificate request for " subject)]
```

Comment:
I think that it is only being logged. Underneath each line with the let, msg is being passed to log/*. @nathanielksmith correct me if I'm wrong.